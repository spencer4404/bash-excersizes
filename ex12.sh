#!/bin/bash

# Filename: ex12.sh
#
# Problem: Make the code into a function and call function using cat, dog 
# and frog as an argument.

functionName(){

	echo The $1 chased it tail.
	echo Then the $1 sat on a $2.
}

functionName 'cat' mat
functionName dog kennel
functionName frog lily  

