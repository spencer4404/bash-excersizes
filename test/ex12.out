The cat chased it tail.
Then the cat sat on a mat.
The dog chased it tail.
Then the dog sat on a kennel.
The frog chased it tail.
Then the frog sat on a lily.
